stubs/protobuf/google/protobuf/internal/enum_type_wrapper.pyi

```diff
@@ -10,7 +10,7 @@ class _EnumTypeWrapper(Generic[_V]):
     DESCRIPTOR: EnumDescriptor
     def __init__(self, enum_type: EnumDescriptor) -> None: ...
     def Name(self, number: _V) -> str: ...
-    def Value(self, name: Union[str, bytes]) -> _V: ...
+    def Value(self, name: Union[Text, bytes]) -> _V: ...
```

Comment:
Nit (not a blocker): We now prefer PEP 604 syntax:

```suggestion
    def Value(self, name: Text | bytes) -> _V: ...
```

Reply:
I decided to apply this before merge, because why not.